store/src/java/com/zimbra/cs/html/owasp/OwaspDefang.java

```diff
@@ -55,12 +55,27 @@ public void defang(InputStream is, boolean neuterImages, Writer out) throws IOEx
     @Override
     public void defang(Reader reader, boolean neuterImages, Writer out) throws IOException {
         String html = CharStreams.toString(reader);
-        String sanitizedHtml = runSanitizer(html, neuterImages);
+        // Temporary fix to check extra comment line & remove
+        String formatHtml = checkExtraCommentLine(html);
```

Comment:
there should be list of tags to be looked for if they are opened & closed, it may be today <!-- might have more in future. We can have a array list to check for such tags.